docs/3.0.0-beta.x/guides/update-version.md

```diff
@@ -0,0 +1,89 @@
+# Update Strapi version
+
+How to upgrade your application to the latest version of Strapi.
+
+## Upgrading your dependencies
+
+Start by upgrading all your strapi package version.
+
+For example moving from `3.0.0-beta.16` to `3.0.0-beta.17`
+
+:::: tabs cache-lifetime="10" :options="{ useUrlFragment: false }"
+
+::: tab "Before - 3.0.0-beta.16" id="3.0.0-beta.16"
+
+```json
+{
+  //...
+  "dependencies": {
+    "strapi": "3.0.0-beta.16",
+    "strapi-admin": "3.0.0-beta.16",
+    "strapi-hook-bookshelf": "3.0.0-beta.16",
+    "strapi-hook-knex": "3.0.0-beta.16",
+    "strapi-plugin-content-manager": "3.0.0-beta.16",
+    "strapi-plugin-content-type-builder": "3.0.0-beta.16",
+    "strapi-plugin-email": "3.0.0-beta.16",
+    "strapi-plugin-graphql": "3.0.0-beta.16",
+    "strapi-plugin-settings-manager": "3.0.0-beta.16",
+    "strapi-plugin-upload": "3.0.0-beta.16",
+    "strapi-plugin-users-permissions": "3.0.0-beta.16",
+    "strapi-utils": "3.0.0-beta.16"
+  }
+}
+```
+
+:::
+
+::: tab "After - 3.0.0-beta.17" id="3.0.0-beta.17"
+
+```json
+{
+  //...
+  "dependencies": {
+    "strapi": "3.0.0-beta.17",
+    "strapi-admin": "3.0.0-beta.17",
+    "strapi-hook-bookshelf": "3.0.0-beta.17",
+    "strapi-hook-knex": "3.0.0-beta.17",
+    "strapi-plugin-content-manager": "3.0.0-beta.17",
+    "strapi-plugin-content-type-builder": "3.0.0-beta.17",
+    "strapi-plugin-email": "3.0.0-beta.17",
+    "strapi-plugin-graphql": "3.0.0-beta.17",
+    "strapi-plugin-settings-manager": "3.0.0-beta.17",
+    "strapi-plugin-upload": "3.0.0-beta.17",
+    "strapi-plugin-users-permissions": "3.0.0-beta.17",
+    "strapi-utils": "3.0.0-beta.17"
+  }
+}
+```
+
+:::
+
+::::
+
+Then run either `yarn install` or `npm install` to install the specified version.
```

Comment:
May add a note here that you may also need to clean your lock files such as `package-lock.json` or `yarn.lock` just by deleting them and let npm or yarn recreate them.

In rare cases I have seen people needing to manually delete their `node_modules` also to clear up some odd cached package version

Reply:
In my case I just added a script to my package.json 

`"clean": "rm -Rf node_modules build .cache",`

Reply:
Since we changed package version in `strict` mode (removed the `^`) normally it's not necessary.
I will add a `note` about that.

Reply:
Yup that works 😃